crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -60,13 +62,11 @@ pub enum Error {
 ///
 /// # Usage
 ///
-/// There are two different styles to configure individual subscribers: Using the sophisticated
-/// [`SettingsBuilder`] or the simplified tuple style for basic configuration. Currently, three
-/// different subscribers are supported: console output, OTLP log export, and OTLP trace export.
+/// ## Tracing Guard
```

Comment:
Both level-2 headings require a "Tracing Guard", so I think this should be renamed:

```suggestion
/// ## Tracing Builder
```

Reply:
I see there is also a "Builder" header below. Maybe they need combining? Or perhaps this "Tracing Guard" section needs moving above the "Usage" section?

Reply:
This section discusses the tracing guard. The sections below describe how to use it.